### ISSUE TITLE
[Docs] Document how to modify the default issue template

### ISSUE THREAD

**Issue by lyz-code** [go-gitea/gitea#14383]
Given a repository with multiple issue templates, when creating a new issue the choose page is shown with "Default" issue template in addition to the desired issue templates .

I haven't found in the docs how to edit or delete that template. Is it an option?

**Comment by lonix1** [comment]
Did you figure it out yet?

I also wonder about this. I'd like to remove it.

**Comment by lyz-code** [comment]
Nope @lonix1, but as it shows the last it's not a big deal, I have 4 other issue types before that, so users are not using the Default much

**Comment by lonix1** [comment]
Yeah for now I just hide it using userstyles. But would be nice to actually remove it.

**Comment by lcnittl** [comment]
> Yeah for now I just hide it using userstyles. But would be nice to actually remove it.

May I ask you exactly this is done?

I totally agree that there should be an option to remove/disable it. I would apparently think that it should be removed by default as soon as custom templates are added.

Cheers!

**Comment by lonix1** [comment]
@lcnittl Just get a userstyles extesion for your browser, e.g. stylus or stylish.

Then write a custom stylesheet for the gitea site and remove that DOM item.

**Comment by karolyi** [comment]
Chiming in here, it would be nice to be able to delete or disable the "default issue template" on a per project basis.

**Comment by karolyi** [comment]
Seemingly, some progress is being made on this: #20956

**Comment by witten** [comment]
The #20956 solution doesn't appear to work for this. Even when setting `blank_issues_enabled` in issue config, the default issue template is still showing up as an option for users to select. Anyone have any ideas?
